enterprise/reporting/reporting.go

```diff
@@ -445,13 +445,17 @@ func transformMetricForPII(metric types.PUReportedMetric, piiColumns []string) t
 			metric.StatusDetail.SampleEvent = []byte(`{}`)
 		} else if col == "sample_response" {
 			metric.StatusDetail.SampleResponse = ""
+		} else if col == "event_name" {
```

Comment:
<img width="641" alt="Screenshot 2022-09-19 at 3 23 42 PM" src="https://user-images.githubusercontent.com/16116691/190992978-79614445-fc20-4251-a257-95f315b4a0d6.png">

Reply:
Left them in the code in case we may need to filter event_name & event_type in the future. Right now, we are not. https://rudderlabs.slack.com/archives/C03MJ8AP78E/p1661884205170179?thread_ts=1661867090.472649&cid=C03MJ8AP78E